### PR TITLE
Local services doc

### PR DESCRIPTION
**Pull request by plauche**
This document discussing configuring and running services in a local environment.

### CODE REVIEW

**Comment by plauche** [comment]
[Getting Started - Running Kubos Core Services Locally](https://trello.com/c/7htBpKyp/953-getting-started-running-kubos-core-services-locally)

File: docs/getting-started/local-services.rst [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+    port = 8000
+
+A local configuration file will only need config data for the services which you
+intend to run locally. All :doc:`core services <../os-docs/services/core-services>`
+provided by Kubos document their configuration options within their respective pages.
+
+Running Services
+----------------
+
+Each service can either be run from the root of the Kubos repo with this command::
```
Comment (jacoffey3) [inline docs/getting-started/local-services.rst]: I would avoid the word "root", use base

File: docs/getting-started/local-services.rst [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+
+The KubOS core services all rely on a central file for run-time configuration information.
+In a running Kubos Linux system that file would be located at 
+``/home/system/etc/config.toml`` and that is where the services will look by default.
+However when running services locally that file may not exist and probably
+will not exist in that location.
+
+This central configuration file is in the ``toml`` format and contains a section
+for each service. The standard config shipped with Kubos Linux can be found
+`here <https://github.com/kubos/kubos-linux-build/blob/master/common/user-overlay/system/etc/config.toml>`__ and is a good reference when building your own configuration file.
```
Comment (jacoffey3) [inline docs/getting-started/local-services.rst]: I think we should have a base config.toml for the core services in our repo, and link to it here.

Then rework the doc to basically say: "Here is the config you can use when running all Core Services..." 

"If you want to run a hardware service, here is the link to how to edit it to run one or more hardware services or a payload service"

File: docs/getting-started/local-services.rst [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+you can use when running the core services. It can be found at ``tools/default_config.toml``
+in the Kubos repo folder.
+
+In the config file each service has their own section(s). Any custom configuration
+for the service will go in a section named after the service::
+
+    [service-name]
+    config_option = "value"
+
+Any service which exposes a GraphQL interface will have a separate section for
```
Comment (CatGarab) [inline docs/getting-started/local-services.rst]: nit: might be worth noting that all core services have an `addr` section

File: docs/getting-started/local-services.rst [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+intend to run locally. If you would like to run a hardware service, each existing
+:doc:`hardware service <../os-docs/services/hardware-services>`
+provided by Kubos documents their configuration options within their respective pages.
+
+Running Services
+----------------
+
+Each service can either be run from the base of the Kubos repo with this command::
+
+    $ cargo run --bin service-name -- -c tools/default_config.toml
```
Comment (CatGarab) [inline docs/getting-started/local-services.rst]: oh neat. TIL

File: docs/getting-started/local-services.rst [hunk 5]
```diff
@@ -1,5 +1,61 @@
-Running Kubos Core Services Locally
+Running KubOS Core Services Locally
 ===================================
 
-- How to configure the services
-- How to run the core services (building should have been taken care of already?)
+This document will lead you through the steps of configuring and running the KubOS core
+services locally. All of the Kubos core services are capable of being run within
```
Comment (CatGarab) [inline docs/getting-started/local-services.rst]: I realize this is entirely my fault, but....
Kubos -> KubOS

**Pull request merged**